### ISSUE TITLE
validate_text raises Net::HTTPRetriableError

### ISSUE THREAD

**Issue by mmb** [alexdunae/w3c_validators#16]
It looks like w3.org is now redirecting.

```
validator.validate_text(rendered)

Net::HTTPRetriableError: 302 "Found"
```

**Comment by GregOnNet** [comment]
Hey there,

first thanks providing w3c_validators.
We use this gem to check our HTML-Markup. We love it!

I also ran into problems running the W3C-Check.

```
Message:

Net::HTTPRetriableError: 302 "Found"

Compilation stack:

  (empty)

Stack trace:

  0. C:/ruby/lib/ruby/2.2.0/net/http/response.rb:119:in `error!'
  1. C:/ruby/lib/ruby/2.2.0/net/http/response.rb:128:in `value'
  2. C:/ruby/lib/ruby/gems/2.2.0/gems/w3c_validators-1.2/lib/w3c_validators/validator.rb:88:in `send_request'
  3. C:/ruby/lib/ruby/gems/2.2.0/gems/w3c_validators-1.2/lib/w3c_validators/validator.rb:85:in `send_request'
  4. C:/ruby/lib/ruby/gems/2.2.0/gems/w3c_validators-1.2/lib/w3c_validators/markup_validator.rb:131:in `validate'
  5. C:/ruby/lib/ruby/gems/2.2.0/gems/w3c_validators-1.2/lib/w3c_validators/markup_validator.rb:117:in `validate_file'
  6. C:/ruby/lib/ruby/gems/2.2.0/gems/nanoc-4.2.4/lib/nanoc/extra/checking/checks/w3c_validator.rb:8:in `block in run'
  7. C:/ruby/lib/ruby/gems/2.2.0/gems/nanoc-4.2.4/lib/nanoc/extra/checking/checks/w3c_validator.rb:7:in `each'
  8. C:/ruby/lib/ruby/gems/2.2.0/gems/nanoc-4.2.4/lib/nanoc/extra/checking/checks/w3c_validator.rb:7:in `run'
  9. C:/ruby/lib/ruby/gems/2.2.0/gems/nanoc-4.2.4/lib/nanoc/extra/checking/runner.rb:120:in `block in run_checks'
  ... 21 more lines omitted. See full crash log for details.
```

More detailed information can be found in the attached crashlog below.

[crash.txt](https://github.com/alexdunae/w3c_validators/files/433059/crash.txt)

It would be great having a solution for this issue.

Thanks in advance
Greg

**Comment by yegor256** [comment]
Same problem here. Any ideas?

**Comment by GregOnNet** [comment]
Hi,

meanwhile I switched to [html-proofer](https://github.com/gjtorikian/html-proofer).
It has a need API.

**Comment by yegor256** [comment]
@mmb I hope this will help: https://github.com/alexdunae/w3c_validators/pull/18

**Comment by rchekaluk** [comment]
Workaround: Instead of this
```validator = MarkupValidator.new```

Use this:
```validator = MarkupValidator.new(:validator_uri => 'http://validator.w3.org/nu/')```

If this is indeed the proper solution, it should be trivial to fix the gem.

**Comment by doc75** [comment]
Normally closed with PR #18.
Let me know if it does not solve it.
